utils/deleteUsers.js

```diff
@@ -0,0 +1,22 @@
+import { createClient } from '@supabase/supabase-js';
+import dotenv from 'dotenv'
+dotenv.config();
+
+//Purpose of this is to remove either fake or real user data with the id of the user to delete. You can only delete from the auth.users table with the external api.
+//This will be converted into an actual rest api route at a later point. This is just a quick work around to not clog up the db in the meantime.
+const deleteUsers = async(userIds) => {

```

Comment:
A delete users function like this is going to be great for testing and getting rid of unneeded data in the database!